include/alpaka/vec/Vec.hpp

```diff
@@ -721,7 +764,7 @@ namespace alpaka
         }
 
         //-----------------------------------------------------------------------------
-        //! \return The element wise less than relation of two vectors.
+        //! \return The element wise less equal than relation of two vectors.
```

Comment:
I think 'element wise' present in many places is not a correct form, should be element-wise or elementwise https://en.wiktionary.org/wiki/elementwise

Also here 'less equal than' should probably be phrased as 'less than or equal'

Reply:
Fixed